client/data/deposits/hooks.ts

```diff
@@ -235,18 +235,18 @@ export const useDepositsSummary = ( {
 };
 
 export const useInstantDeposit = (
-	transactionIds: string[]
+	currency: string
 ): { inProgress: boolean; submit: () => void; deposit: unknown } => {
```

Comment:
Thought (out of scope of this PR): we should use an interface from `deposits.d.ts` here.

However, low priority, since this value from this hook is actually not currently used in our codebase (we only use `inProgress` and `submit`).

Reply:
I was wondering about that actually, whether it's necessary to return `deposit` since it's not used anywhere.